sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample2_DetectLastPointAnomaly.cs

```diff
@@ -49,17 +49,29 @@ public async Task DetectLastPointAnomaly()
             //detect
             Console.WriteLine("Detecting the anomaly status of the latest point in the series.");
 
-            LastDetectResponse result = await client.DetectLastPointAsync(request).ConfigureAwait(false);
+            try
+            {
+                LastDetectResponse result = await client.DetectLastPointAsync(request).ConfigureAwait(false);
 
-            if (result.IsAnomaly)
+                if (result.IsAnomaly)
+                {
+                    Console.WriteLine("The latest point was detected as an anomaly.");
+                }
+                else
+                {
+                    Console.WriteLine("The latest point was not detected as an anomaly.");
+                }
+            }
+            catch (RequestFailedException ex)
             {
-                Console.WriteLine("The latest point was detected as an anomaly.");
+                Console.WriteLine(String.Format("Last detection failed: {0}", ex.Message));
+                throw new Exception(ex.Message);
             }
-            else
+            catch (Exception ex)
             {
-                Console.WriteLine("The latest point was not detected as an anomaly.");
+                Console.WriteLine(String.Format("Detection error. {0}", ex.Message));
+                throw new Exception(ex.Message);
```

Comment:
```suggestion
                throw;
```

Wrapping the exception will both change the underlying type, making it harder for callers to handle, and will reset the stack trace, forcing callers to dig into the inner exception.   I'd recommend simply rethrowing here to preserve the original exception information.

Reply:
Updated all similar Exception throw as suggested.